vault/resource_mount_test.go

```diff
@@ -140,6 +140,28 @@ func TestResourceMount_KVV2(t *testing.T) {
 	})
 }
 
+func TestResourceMount_ExternalEntropyAccess(t *testing.T) {
+	path := acctest.RandomWithPrefix("example")
+	resource.Test(t, resource.TestCase{
+		Providers: testProviders,
+		PreCheck:  func() { testAccPreCheck(t) },
+		Steps: []resource.TestStep{
+			{
+				Config: testResourceMount_InitialConfigExternalEntropyAccess(path),
+				Check:  testResourceMount_CheckExternalEntropyAccess(path, false),
+			},
+			{
+				Config: testResourceMount_UpdateConfigExternalEntropyAccess(path, true),
+				Check:  testResourceMount_CheckExternalEntropyAccess(path, true),
+			},
+			{
+				Config: testResourceMount_UpdateConfigExternalEntropyAccess(path, false),
+				Check:  testResourceMount_CheckExternalEntropyAccess(path, false),
+			},
```

Comment:
Could you add another two `TestStep` entries here, the first using `testResourceMount_UpdateConfigExternalEntropyAccess(path, true),` and the second using `testResourceMount_InitialConfigExternalEntropyAccess(path)`, and verify that `mount.ExternalEntropyAccess` is false? I'd like to ensure that if `external_entropy_access` is initially set to `true`, and then later completely removed from the config (as opposed to set to `false`) that we do the right thing.

Reply:
Good idea! I've added those two additional `TestStep` entries in 51712d7. I also manually tested out going from `external_entropy_access = true` to completely removing it to ensure the configuration on the mount is what's expected (false).

Test rerun:
```
$ TESTARGS='-run=TestResourceMount -count=1' make testacc                     
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test $(go list ./... |grep -v 'vendor') -v -run=TestResourceMount -count=1 -timeout 120m
?       github.com/terraform-providers/terraform-provider-vault [no test files]
?       github.com/terraform-providers/terraform-provider-vault/cmd/coverage    [no test files]
testing: warning: no tests to run
PASS
ok      github.com/terraform-providers/terraform-provider-vault/util    0.471s [no tests to run]
=== RUN   TestResourceMount
--- PASS: TestResourceMount (0.18s)
=== RUN   TestResourceMount_Local
--- PASS: TestResourceMount_Local (0.16s)
=== RUN   TestResourceMount_SealWrap
--- PASS: TestResourceMount_SealWrap (0.16s)
=== RUN   TestResourceMount_KVV2
--- PASS: TestResourceMount_KVV2 (0.13s)
=== RUN   TestResourceMount_ExternalEntropyAccess
--- PASS: TestResourceMount_ExternalEntropyAccess (0.35s)
PASS
ok      github.com/terraform-providers/terraform-provider-vault/vault   1.420s

```

Reply:
👍